src/test/kotlin/mate/academy/HelloWorldAnonymousClassesTest.kt

```diff
@@ -1,5 +1,6 @@
 package mate.academy
 
+import HelloWorldAnonymousClasses
```

Comment:
don`t make changes in tests

Reply:
not fixed - please undo
review your own PR to make sure this file isn't here